lib/simplecov/merge_helper.rb

```diff
@@ -0,0 +1,31 @@
+module SimpleCov
+  module MergeHelper
+    class << self
+      # Merges two Coverage.result hashes
+      def merge_resultsets(a, b)
```

Comment:
I think this is a long needed feature. Any reason it shouldn't just be a function on the ResultMerger?